pkgs/desktops/pantheon/apps/elementary-code/default.nix

```diff
@@ -71,12 +64,14 @@ stdenv.mkDerivation rec {
     elementary-icon-theme
     granite
     gtk3
-    gtksourceview3
+    gtksourceview4
     gtkspell3
     libgee
     libgit2-glib
+    libhandy
     libpeas
     libsoup
+    polkit
```

Comment:
Could you move this to `nativeBuildInputs` and add a comment that this is for ITS rules? Otherwise looks good.

Reply:
Done.